### DISCUSSION TITLE
Sumatra does not remember the path to the flash drives

### DISCUSSION THREAD

**Discussion by Tenedoris** [sumatrapdfreader/sumatrapdf#2812] [General]
As of version 3.2, Sumatra does not remember the path to the flash drives. Sumatra writes a question mark in place of the drive letter, e.g.   ?:\My books\.
Is it a mistake or a deliberate action?

**Comment by GitHubRulesOK** [comment]
it was a deliberate means to avoid some issues, but behavior has changed several times since then, so best to compare against current pre-release
